python/grass/jupyter/timeseries.py

```diff
@@ -0,0 +1,87 @@
+# MODULE:    grass.jupyter.timeseries
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains functions visualizing time-space raster
+#            and vector datasets in Jupyter Notebooks
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#           This program is free software under the GNU General Public
+#           License (>=v2). Read the file COPYING that comes with GRASS
+#           for details.
+
+import tempfile
+import grass.script as gs
+
+from .region import RegionManagerFor2D, RegionManagerFor3D
+import ipywidgets as widgets
+
+
+class timeseries:
+    """timeseries creates visualization of time-space raster and
+    vector dataset in Jupyter Notebooks"""
+
+    def __init__(self, map, basemap, overlay):
+        self.map = map
+        self.basemap = basemap
+        self.overlay = overlay
+        # Check that map ends with tsrds or tsvds
```

Comment:
The abbreviations we use are: strds, stvds and str3ds. I'd suggest to keep it consistent if possible.

Reply:
Oops! Thank you for catching this. I'll make sure they are consistent.

Reply:
I'm afraid that in the Python API for Jupyter, the abbreviations won't look really good and won't fit with the rest in terms of readability. Suggestions welcome.

Reply:
My point was to use the same terms we use in the temporal framework as letters were exchanged and that might create confusion. 

Sorry for my ignorance here but, what do you mean by "in the Python API for Jupyter, the abbreviations won't look really good and won't fit with the rest"? Not using any abbreviations at all? Or finding new abbreviations to refer to strds and such? We can say raster time series and vector time series for example. Still, in the notebook or documentation I'd clarify that they refer to the temporal framework objects known as space-time raster dataset and such... If that's indeed what will be used in this context of course. Otherwise, ignore the noise :)

Reply:
> ...letters were exchanged...

Yes, right, that's correct.

> ...Not using any abbreviations at all?...

Maybe. The tsrds and tsvds in the code just looks as arbitrary as the strds-stvds-str3ds mumble jumble. In grass.jupyter, the idea is that the names fit with Python naming convention unless there is a reason not to, so _d.rast_ could be hidden under _display_raster_, but "d-rast" is pronounceable and you can match it with "display raster" or it is at least easy enough to document and definitively it is easy to document that the _d_rast_ function is the _d.rast_ tool/module, so at the end, _d_rast_ looks like a descriptive, but pragmatically shortened name in Python (like e.g. _os.environ_ or _sys.path_). It now seems to me that with temporal datasets, we need to have similar discussion we had before for d-commands in grass.jupyter.